cmake/ModuleHelpers.cmake

```diff
@@ -10,13 +12,33 @@ macro(_userver_module_begin)
     set(multiValueArgs
         DEBIAN_NAMES FORMULA_NAMES RPM_NAMES PACMAN_NAMES PKG_NAMES
         # For version detection of manually installed packages and unknown package managers.
-        PKG_CONFIG_NAMES
+	PKG_CONFIG_NAMES
+	# For CPM options
+	CPM_NAME
+        CPM_VERSION
+        CPM_GITHUB_REPOSITORY
+        CPM_OPTIONS
+        CPM_SOURCE_SUBDIR
+        CPM_GIT_TAG
     )
 
     cmake_parse_arguments(ARG "${options}" "${oneValueArgs}" "${multiValueArgs}" "${ARGN}")
 
     set(name "${ARG_NAME}")
 
+    if(ARG_CPM_NAME)
+        option(
+            USERVER_DOWNLOAD_PACKAGE_${ARG_CPM_NAME}
```

Comment:
This creates vars like `USERVER_DOWNLOAD_PACKAGE_c-ares`. There should probably be an extra package name for userver cmake variables to keep variable naming sane?

Reply:
fixed